composer.json

```diff
@@ -17,6 +17,11 @@
         "phpunit/phpunit": "^6.5.5 || ^7.0.0",
         "wp-coding-standards/wpcs": "^0.14.0 || ^1.0.0 || ^2.0.0",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.7.1"
+    },
+	"config": {
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
+        },
     },
```

Comment:
```suggestion
    "config": {
        "allow-plugins": {
            "dealerdirect/phpcodesniffer-composer-installer": true
        }
    },
```

Reply:
fixed in https://github.com/wp-cli/wp-config-transformer/commit/8854552236c8838ea3caa4f6a62c5901af84543b